js/scripts.js

```diff
@@ -35,6 +35,7 @@ var app = {
         }, 500);
 
       app.updateURL(url);
+      $('input').val('');
```

Comment:
@nwerbeckes-gpsw yes we need both. line 8 clears the input value upon first view. line 38 clears the input value after successfully showing the html file, every time.